### ISSUE TITLE
Too much slow when posting a photo

### ISSUE THREAD

**Issue by jsfan3** [shannah/RADChatRoom#17]
Trying your demo Android apk on my Android 7 device, the app is completely frozen for some seconds to load a photo.

My hypothesis is that your code resizes the photo and it needs some seconds, blocking the EDT.

**Comment by shannah** [comment]
Yes.  The source of the demo is [here](https://github.com/shannah/RADChatRoom/blob/master/CN1Chat/src/com/codename1/cn1chat/ChatFormController.java#L152)

As it was the demo, I just did it the "default" way.  However, this is probably something to look at in CN1 core.

Probably I should be limiting the width of the photo in the `Capture.capturePhoto()` call to prevent it from taking a high-res photo.
